zombies.js

```diff
@@ -158,6 +233,20 @@
  * @name equip
  * @param {Weapon} itemToEquip  The weapon item to equip.
  */
+equip(itemsToEquip) {
+  let index = this._pack.indexOf(itemsToEquip);
+  
+  if(itemsToEquip instanceof Weapon && index !== -1) {
+    if(!this.equipped) {
```

Comment:
I recommend flipping the if and the else to avoid using ! in your if conditionals. It makes the code easier to read.